package/crds/sql.azure.jet.crossplane.io_servers.yaml

```diff
@@ -127,6 +127,8 @@ spec:
                     type: array
                   location:
                     type: string
+                  name:
```

Comment:
Do you know why this came back?

Reply:
I will try to push a fix for this. Thank you @muvaf.